src/main/kotlin/no/fdk/fdk_reasoning_service/config/CircuitBreakerConsumerConfiguration.kt

```diff
@@ -0,0 +1,53 @@
+package no.fdk.fdk_reasoning_service.config
+
+import io.github.resilience4j.circuitbreaker.CircuitBreaker.StateTransition
+import io.github.resilience4j.circuitbreaker.CircuitBreakerRegistry
+import io.github.resilience4j.circuitbreaker.event.CircuitBreakerOnStateTransitionEvent
+import no.fdk.fdk_reasoning_service.kafka.KafkaManager
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+open class CircuitBreakerConsumerConfiguration(
+    private val circuitBreakerRegistry: CircuitBreakerRegistry,
+    private val kafkaManager: KafkaManager
+) {
+    init {
+        LOGGER.debug("Configuring circuit breaker event listener")
+        val topics = listOf(
```

Comment:
For det første er det ikke topics men listeners du behandler her. For det andre er det kun 1 listener og den har id "reasoning".

Sjekk id Kafka listener.